### PR TITLE
Str exercise 06

### PR DESCRIPTION
**Pull request by trandiepphuong**

### CODE REVIEW

File: src/main/java/StrEx6.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    public String rutgon(String n) {
+        n = n + " ";
+        String newStr = "";
+        int count = 1;
+        for (int i = 1; i < n.length(); i++) {
+            if (n.charAt(i) == n.charAt(i - 1)) {
+                count++;
+            } else {
+                if (count != 1) {
+                    newStr += n.charAt(i - 1) + String.valueOf(count);
```
Comment (voquanghoa) [inline src/main/java/StrEx6.java:16]:
```suggestion
                    newStr += n.charAt(i - 1) + "" + count;
```

File: src/main/java/StrEx6.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+                } else {
+                    newStr += n.charAt(i - 1);
+                }
+                count = 1;
+            }
```
Comment (voquanghoa) [inline src/main/java/StrEx6.java]:
```suggestion
            } else {
                newStr += n.charAt(i - 1);
                if (count != 1) {
                    newStr += count;
                }
                count = 1;
            }
```

Comment (trandiepphuong) [reply src/main/java/StrEx6.java]: Done

File: src/main/java/StrEx6.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class StrEx6 {
+    public String rutgon(String n) {
+        n = n + " ";
+        String newStr = "";
```
Comment (voquanghoa) [inline src/main/java/StrEx6.java]: Dùng StringBuffer

Comment (trandiepphuong) [reply src/main/java/StrEx6.java]: StringBuilder chu a?

Comment (trandiepphuong) [reply src/main/java/StrEx6.java]: Done

Comment (trandiepphuong) [reply src/main/java/StrEx6.java]: Done

**Pull request merged**